test/performance/specs/front-end-block-theme.spec.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+/**
+ * Internal dependencies
+ */
+const { saveResultsFile } = require( '../utils' );
+
+const results = {
+	timeToFirstByte: [],
+	largestContentfulPaint: [],
+	lcpMinusTtfb: [],
+};
```

Comment:
I think it might be a good opportunity to use the [`attach`](https://playwright.dev/docs/api/class-testinfo#test-info-attach) API here? It also work pretty nicely with custom reporters.

Reply:
Sounds good! I'd also leave that for a follow-up PR, though, as it might involve more changes that would be nice to track separately.